bakerydemo/jinja2/base/include/header-blog.html

```diff
@@ -19,7 +17,7 @@ <h1 class="index-header__title index-header__title--blog">{{ page.title }}</h1>
             {% endif %}
             {% if page.date_published %}
                 <div class="blog__published">
-                    {{ page.date_published }}
+                    {{ page.date_published|date("DATE_FORMAT") }}
```

Comment:
I’m surprised Jinja doesn’t do this by default, perhaps there’s a way with configuration.